internal/namespaces/instance/v1/securitygroup_test.go

```diff
@@ -0,0 +1,97 @@
+package instance
+
+import (
+	"testing"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+)
+
+func Test_SecurityGroupClear(t *testing.T) {
+
+	t.Run("Usage", core.Test(&core.TestConfig{
```

Comment:
we should remove this test

all usage are tested in main

see https://github.com/scaleway/scaleway-cli/blob/v2/cmd/scw/main_test.go